src/sidebar/components/MarkdownEditor.js

```diff
@@ -138,20 +144,52 @@ function ToolbarButton({
 
   if (label) {
     return (
-      <LabeledButton classes="font-normal TransparentButton" {...buttonProps}>
+      <LabeledButton
+        classes={classNames(
+          'font-normal bg-transparent',
+          // TODO: Refactor shared button styles to reduce specificity and make
+          // this !important rule unnecessary
```

Comment:
What refactoring are you planning to do here to make this unnecessary?

Reply:
Same as the last time this came up in a PR this week; styling rules for shared button components have too high of specificity.